### DISCUSSION TITLE
Full stops in sentences inside namespace?

### DISCUSSION THREAD

**Discussion by CzechCoder** [amannn/next-intl#1205] [Q&A]
When I want to translate whole set of sentences i.e. some info text like "One sentence. Two sentence. Three sentence.", the translations suddenly stop working, telling me to remove the "." character because of nesting. Is there a way to overcome this without having to edit all the namespaces? I have a json that I received from my client and editing it would be very tedious and time consuming.

**Comment by thierry-rietsch** [comment]
Second that. For me, having the option to use real text values as keys is something that I really would appreciate.
